### DISCUSSION TITLE
Using a gradient boosted model instead of transformer for inference?

### DISCUSSION THREAD

**Discussion by asenzz** [ggml-org/llama.cpp#17865] [Ideas]
Have you considered using a GBM such is LightGBM or XGBoost instead of Transformer to improve accuracy?
